hooks/00-extra-packages.chroot

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
-#
 set -eux
 
 apt update
+apt dist-upgrade -y
```

Comment:
Aha, we need this to avoid extra bootstraps when iterating?

Reply:
Not really, I just added it felt right.